### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by shivert**

### CODE REVIEW

File: lib/greenhouse_io/api/client.rb [hunk 1]
```diff
@@ -32,10 +32,26 @@ def activity_feed(id, options = {})
...
+      )
+    end
+
+    def create_candidate(body, on_behalf_of)
+      post_to_harvest_api(
+        "/candidates",
+        body,
+        { 'On-Behalf-Of' => on_behalf_of.to_s }
+      )
+    end
```
Comment (DanFan1988) [inline lib/greenhouse_io/api/client.rb:49]: :cool:

File: lib/greenhouse_io/api/client.rb [hunk 2]
```diff
@@ -32,10 +32,26 @@ def activity_feed(id, options = {})
       get_from_harvest_api "/candidates/#{id}/activity_feed", options
     end
 
-    def create_candidate_note(candidate_id, note_hash, on_behalf_of)
+    def create_candidate_note(candidate_id, body, on_behalf_of)
```
Comment (DanFan1988) [inline lib/greenhouse_io/api/client.rb:35]: this is literally the best line of code i've ever seen

File: lib/greenhouse_io/api/client.rb [hunk 3]
```diff
@@ -84,8 +100,6 @@ def sources(id = nil, options = {})
       get_from_harvest_api "/sources#{path_id(id)}", options
     end
 
-    private
-
```
Comment (DanFan1988) [inline lib/greenhouse_io/api/client.rb:88]: r u nsa

**Review by DanFan1988 — APPROVED** [review]

File: lib/greenhouse_io/api/client.rb (hunk 3 above)
Comment (shivert) [reply lib/greenhouse_io/api/client.rb:88]: 🇺🇸

**Pull request merged**